main.go

```diff
@@ -14,12 +14,12 @@ import (
 	"path"
 )
 
-type GcloudKeyFile struct {
+type gcloudKeyFile struct {
```

Comment:
Should start with uppercase `G`, otherwise the struct can't be serialized into JSON